Menus/RadialMenu/Scripts/RadialMenuUI.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace UnityEditor.Experimental.EditorVR.Menus
 {
-	sealed class RadialMenuUI : MonoBehaviour, IConnectInterfaces
+	sealed class RadialMenuUI : MonoBehaviour, IConnectInterfaces, IPerformHaptics
```

Comment:
It should; and I originally implemented it in that manner.  Though, the rayOrigin was still null after connecting interfaces on it, so I went with setting the rayOrigin via the RadialMenu, after instantiating the UI.  I'll look into the potential cause of such a thing happening.